src/configuration/config.go

```diff
@@ -71,6 +71,8 @@ type DeviceConfig struct {
 	SSHAuthMethod    string
 	SSHPrivateKey    string
 	AllowInsecureSSH bool
+	// Route53 settings
+	ZoneID string
```

Comment:
The only thing that nags at me is having a device specific configuration in the master config design. Makes me wonder if we need some sort of generic "parameter1" "parameter2" type system. But, another approach is to let this go and see how many other device types need device specific configs, then try to generalize it later if it gets out of hand.